core/src/test/java/com/amplifyframework/core/model/ModelSchemaTest.java

```diff
@@ -45,9 +45,10 @@
      * The factory {@link ModelSchema#fromModelClass(Class)} will produce
      * an {@link ModelSchema} that meets our expectations for the {@link Person} model.
      * @throws AmplifyException from model schema parsing
+     * @throws ClassNotFoundException from Class.forName("com.amplifyframework.core.model.temporal.Temporal$Date")
      */
     @Test
-    public void modelSchemaIsGeneratedForPersonModel() throws AmplifyException {
+    public void modelSchemaIsGeneratedForPersonModel() throws AmplifyException, ClassNotFoundException {
```

Comment:
The `ClassNotFoundException` has gone away now, too.

Reply:
Just fixed!